aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,8 +1,301 @@
-.. include:: /../common/stub-topic.txt
+Bundling and Minification
+=========================
 
-|stub-icon| Bundling and Minification
-=====================================
+By `Erik Reitan`_ 
 
-.. include:: /../common/stub-notice.txt
+Bundling and minification are two techniques you can use in ASP.NET 5 to improve page load performance for your web application. Bundling makes it easy to combine or bundle multiple files into a single file. Minification performs a variety of different code optimizations to scripts and CSS, which results in smaller file sizes. Bundling and minification improves load time performance by reducing the number of requests to the server and reducing the size of the requested assets (such as CSS and JavaScript files.)
 
-.. _issue: https://github.com/aspnet/Docs/issues/101
+This article explains and shows how to use the bundling and minification features included with the ASP.NET 5 Web Application template in `Visual Studio 2015 <http://go.microsoft.com/fwlink/?LinkId=517106>`_. 
+
+Overview
+--------
+Because bundling combines multiple files into a single file, it reduces the number of requests to the server that is required to retrieve and display a web asset, such as a web page. You can create CSS, JavaScript and other bundles. Fewer files, means fewer HTTP requests from your browser to the server or from the service providing your application. This results in improved first page load performance.
```

Comment:
Fewer files, means fewer HTTP requests from your browser to the server or from the service providing your application. This results in improved first page load performance. 
Not necessarily. From my original (you'll need to update the 6)Most of the current major browsers limit the number of simultaneous connections per each hostname to six. That means that while six requests are being processed, additional requests for assets on a host will be queued by the browser. In the image below, the IE F12 developer tools network tabs shows the timing for assets required by the About view of a sample application.